.teamcity/_self/projects/WPComPlugins.kt

```diff
@@ -61,6 +63,54 @@ object WPComPlugins : Project({
 	}
 })
 
+object BuildPlugins: BuildType({
+	id("calypso_WPComPlugins_Build_Plugins")
+	uuid = "8453b8fe-226f-4e91-b5cc-8bdad15e0814"
+	name = "Build WPCom Plugins"
```

Comment:
While we have the chance, I'd like to rework the naming to "calypso apps" instead of plugins. I think plugins doesn't accurately describe the projects (most of them _aren't_) plugins, whereas we've used "calypso app" in a lot of places before

Reply:
I think we could also move it more to the root level instead of the weird nested project we have now (both in the file directory and in teamcity), but that could probably be a follow up?

Reply:
> I think we could also move it more to the root level instead of the weird nested project we have now (both in the file directory and in teamcity), but that could probably be a follow up?

Let's do that as a follow-up once the migration work is complete.

> rework the naming to "calypso apps" instead of plugins
Sure, I'll rename this `Calypso Apps`.